### ISSUE TITLE
[prometheus-postgres-exporter] Upgrade instructions to 5.0.0 do not accurately warn the user of the severity of missing custom queries.

### ISSUE THREAD

**Issue by sig-piskule** [prometheus-community/helm-charts#3792]
### Describe the bug a clear and concise description of what the bug is.

The readme file here does not inform the user well enough of the impact of the deprecation of custom queries.

https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus-postgres-exporter/README.md

I had to analyze the Helm Chart code as well as the Postgres Exporter docker image to gain an accurate understanding of the problem. It appeared as though `config.queries` was simply renamed to `config.postgresExporter`, and that I needed to alter my queries to a new `postgres_exporter.yml` format. However, that is not the case.

`config.queries` is totally removed, and that is a major breaking change. That actually removes a core feature that I use this helm chart for. The upgrade to 5.0.0 additionally does not suggest alternatives for how to fix this. The docker image suggests using, https://github.com/burningalchemist/sql_exporter. but at the time of my writing I am unsure if there is a different helm chart to use this.

I would have hoped that a major warning about a total feature removal would have been apparent. My opinion is that something in the Readme along the lines of "If you use `config.queries`, this feature is no longer supported, and you need to use a different helm chart" should have been somewhere in the readme.

### What's your helm version?

v3.8.1

### What's your kubectl version?

v1.18.2

### Which chart?

prometheus-community/prometheus-postgres-exporter

### What's the chart version?

5.0.0

### What happened?

I am unable to upgrade to 5.0.0 due to feature removal.

### What you expected to happen?

There would have been some major warning that a core feature is no longer supported.

### How to reproduce it?

N/A

### Enter the changed values of values.yaml?

```
 config:
-  userQueries: |-
+  postgresExporter: |-
```

### Enter the command that you execute and failing/misfunctioning.

N/A

```
postgres_exporter: error: unknown long flag '--config.file', try --help
```

### Anything else we need to know?

Part of the reason why I am entering this is because this was not visible anywhere on Google / Documentation that `postgres_exporter.yml` / `config.postgresExporter` is not the same as `config.queries`.

There is no upgrade path from 4.X to 5.0 for us.

Issue Link: https://github.com/prometheus-community/postgres_exporter/issues/909

**Comment by hajdukda** [comment]
Just hit the same issue :D... The default scraping is terrible, the cardinality is so bad we had to override default queries, now we have no option to do that.
TL;DR is that 5.0.0 is unusable in any moderate setup.
